src/Collector/IndexCollectorInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Yiisoft\Yii\Debug\Collector;
+
+/**
+ * Index data collector responsibility is to collect data during application lifecycle.
+ */
+interface IndexCollectorInterface
+{
+    /**
+     * @return array data indexed
+     */
+    public function getIndexed(): array;
```

Comment:
That is significantly different from EventCollectorInterface. Is that intended?

Reply:
Could be renamed to `getIndexData` or `collectIndexData`.

Reply:
> That is significantly different from EventCollectorInterface. Is that intended?

Yes, there is only the number of events that can be indexed.

> Could be renamed to getIndexData or collectIndexData.

Renamed to `getIndexData`.